Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
But those Twig extensions in SonataMediaBundle don't implement `ExtensionInterface`, but they extend `Twig_Extension` as recommended:

```php
class MediaExtension extends \Twig_Extension implements \Twig_Extension_InitRuntimeInterface
{
    ...
}
```

I tested this with Twig 1.31.0 and didn't have any problems (AFAIK). May be I'm not seeing the big picture.